src/main/java/io/confluent/connect/jdbc/source/BulkTableQuerier.java

```diff
@@ -30,14 +30,16 @@
 
 import io.confluent.connect.jdbc.util.JdbcUtils;
 
+import static io.confluent.connect.jdbc.source.JdbcSourceConnectorConfig.NumericMapping;
+
 /**
  * BulkTableQuerier always returns the entire table.
  */
 public class BulkTableQuerier extends TableQuerier {
   private static final Logger log = LoggerFactory.getLogger(BulkTableQuerier.class);
 
   public BulkTableQuerier(QueryMode mode, String name, String schemaPattern,
-                          String topicPrefix, boolean mapNumerics) {
+                          String topicPrefix, NumericMapping mapNumerics) {
```

Comment:
Since you've updated parameter's type from `boolean` to `NumericMapping ` - maybe change parameter name as well, to something like `numbericMappingType`?

Reply:
Keeping the same name leaves some lines unchanged. Which is what I aim for with this PR. Doesn't sound too irrelevant anyways.